tensordict/nn/distributions/composite.py

```diff
@@ -223,16 +221,32 @@ def rsample(self, shape=None) -> TensorDictBase:
         )
 
     def log_prob(
-        self, sample: TensorDictBase, *, aggregate_probabilities: bool | None = None
+        self,
+        sample: TensorDictBase,
+        *,
+        aggregate_probabilities: bool | None = None,
+        include_sum: bool | None = None,
+        inplace: bool | None = None,
     ) -> torch.Tensor | TensorDictBase:  # noqa: D417
-        """Computes and returns the summed log-prob.
+        """Compute the summed log-probability of a given sample.
 
         Args:
-            sample (TensorDictBase): the sample to compute the log probability.
+            sample (TensorDictBase): The input sample to compute the log probability for.
 
         Keyword Args:
             aggregate_probabilities (bool, optional): if provided, overrides the default ``aggregate_probabilities``
                 from the class.
+            include_sum (bool, optional): Whether to include the summed log-probability in the output TensorDict.
+                Defaults to ``self.inplace`` which is set through the class constructor (``True`` by default).
```

Comment:
```
              Defaults to ``self.include_sum`` which is set through the class constructor (``True`` by default).
```

Reply:
ok correcting these typos in follow-up PR